mmdt/mmdt/urls.py

```diff
@@ -14,6 +14,7 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
```

Comment:
If you didn't change anything in this file, you shouldn't include this change in this PR